### PR TITLE
Fix clicks on custom prescription when editing medicines

### PR DESCRIPTION
**Pull request by sanchita**
https://www.pivotaltracker.com/story/show/172784091

### CODE REVIEW

File: app/src/main/java/org/simple/clinic/drugs/selection/entry/CustomPrescribedDrugListItem.kt [hunk 1]
```diff
@@ -23,17 +23,18 @@ data class CustomPrescribedDrugListItem(
   override fun getLayout() = R.layout.list_prescribeddrugs_custom_drug
 
   override fun createViewHolder(itemView: View): DrugViewHolder {
-    val holder = DrugViewHolder(itemView)
-    holder.itemView.setOnClickListener {
-      uiEvents.onNext(CustomPrescriptionClicked(prescription))
-    }
-    return holder
```
Comment (vinaysshenoy) [inline app/src/main/java/org/simple/clinic/drugs/selection/entry/CustomPrescribedDrugListItem.kt:30]: Wow, how was this missed? 😂

Comment (sanchita) [reply app/src/main/java/org/simple/clinic/drugs/selection/entry/CustomPrescribedDrugListItem.kt:30]: I know, it is kind of hilarious!

**Comment by vinaysshenoy** [comment]
@sanchita do we need to update the `CHANGELOG`?

**Comment by sanchita** [comment]
> @sanchita do we need to update the `CHANGELOG`?

Was just doing that.

**Pull request merged**